packages/api/src/external/fhir/document/index.ts

```diff
@@ -144,3 +156,110 @@ export function idToFHIR(id: DocumentIdentifier): Identifier {
     use: id.use === "unspecified" ? undefined : id.use,
   };
 }
+
+/**
+ * Takes a CW resource and converts it to a FHIR resource
+ *
+ * @param resource
+ * @returns
+ */
+function convertToFHIRResource(resource: Contained): Resource | undefined {
+  if (resource.resourceType === ResourceType.Patient && resource.id) {
+    return {
+      resourceType: ResourceType.Patient,
+      id: resource.id,
+    };
+  } else if (resource.resourceType === ResourceType.Organization && resource.name) {
+    return {
+      resourceType: ResourceType.Organization,
+      name: convertCWNameToString(resource.name),
+    };
+  } else if (resource.resourceType === ResourceType.Practitioner && resource.name) {
+    return {
+      resourceType: ResourceType.Practitioner,
+      name: convertCWNameToHumanName(resource.name),
+    };
+  }
+}
+
+/**
+ * Converts a CW name to a string
+ *
+ * @param name
+ * @returns
+ */
+function convertCWNameToString(name: string | CWHumanName | CWHumanName[]): string | undefined {
+  if (typeof name === "string") {
+    return name;
+  } else if (Array.isArray(name)) {
+    if (name.length) {
+      const names: string[] = [];
+      name.forEach(n => {
+        if (typeof n === "string") {
+          names.push(n);
+        } else if (typeof n === "object") {
+          const humanNames = getHumanNamesFromObject(n).map(n => n.family);
+          if (humanNames) names.push(humanNames.join(" "));
+        }
+      });
+      return names.join(" ");
+    }
+  } else if (typeof name === "object") {
+    return getHumanNamesFromObject(name)
+      .map(n => n.family)
+      .join(" ");
+  }
+}
+
+/**
+ * Converts a CW name to a FHIR HumanName
+ *
+ * @param name
+ * @returns
+ */
+function convertCWNameToHumanName(
+  name: string | CWHumanName | CWHumanName[]
+): HumanName[] | undefined {
+  if (typeof name === "string") {
+    return [
+      {
+        family: name,
+      },
+    ];
+  } else if (Array.isArray(name)) {
+    if (name.length) {
+      let names: HumanName[] = [];
```

Comment:
we're not returning `names`?

Reply:
Oh woops! Good catch!

Reply:
this is marked as resolved but the code doesn't seem updated, maybe forgot to push to remote?